dev/prod_builders.json

```diff
@@ -24,6 +24,12 @@
          "task_name": "linux_android_obfuscate_test",
          "flaky": false
       },
+      {
+         "name": "Linux android_stack_size_test",
+         "repo": "flutter",
+         "task_name": "linux_android_stack_size_test",
+         "flaky": false
```

Comment:
I think you should mark it flaky and make sure it passes before letting it close the tree.